apps/test/unit/testUtilsTest.js

```diff
@@ -63,16 +65,49 @@ describe('forEveryBooleanPermutation', function () {
 describe('throwOnConsoleErrors', function () {
   describe('without it', function () {
     it('console.error does not throw an exception', function () {
-        expect(() => console.error('This console.error call is intentional.'))
-            .not.to.throw();
+      expect(() => console.error('This console.error call is intentional.'))
+          .not.to.throw();
     });
   });
 
   describe('with it', function () {
     throwOnConsoleErrors();
     it('console.error does throw an exception', function () {
-      expect(() => console.error('should throw'))
-          .to.throw(Error, 'should throw');
+      // Suppress excessive logging during test
+      sinon.stub(console, 'log');
```

Comment:
If we're going to run this manually, we might as well let it log excessively (easier to verify that it's working).  Can remove stub/restore from both tests, and then remove the `sinon` dependency in this file.